docs/eigenda/integrations-guides/dispersal/api-documentation/overview.md

```diff
@@ -35,17 +35,29 @@ The Disperser offers an asynchrounous API for dispersing blobs, where clients sh
 
 ## Endpoints
 
-### Dispersal Endpoints
+Here we provide a narrative-level description of the major API endpoints. Please see [the repo](https://github.com/Layr-Labs/eigenda/tree/master/api/docs) for detailed, field-level API documentation.
 
-Blobs can be dispersed through `DisperseBlobAuthenticated()` or `DispersedBlob()` endpoints. The `DispersedBlob()` is used for easy testing of essential functionality in testing environments, sometimes with IP-based whitelists for increased throughput. All production traffic will eventually be required to use the DisperseBlobAuthenticated endpoint. Both endpoints accept a [DisperseBlobRequest](https://github.com/Layr-Labs/eigenda/blob/4d19b0597d1151a402e328430a0ff5345a831fb6/api/proto/disperser/disperser.proto#L72) message, with the following fields:
+### DisperseBlob()
+
+:::info
+This endpoint will likely be deprecated in future releases. All production traffic should prefer the `DisperseBlobAuthenticated` endpoint. 
```

Comment:
we may need a more sure answer to users than "likely", maybe drop it if it's undecided